.github/workflows/stale.yml

```diff
@@ -57,14 +59,20 @@ jobs:
 
           # Labels on issues exempted from stale.
           exempt-issue-labels: |
-            "Status: Blocked,Status: Decision Required,Peloton 🚴‍️"
+            "Status: Blocked,Status: Decision Required,Peloton 🚴‍♂️"
```

Comment:
I appear to have not originally copied the peloton label correctly :blush:

Reply:
I genuinely can't see what the difference is here 😕

Reply:
Looks like the emoji difference visible on Linux but not Windows. This isn't the first time that confusion has come up - worth keeping in mind!

Reply:
It's the *gender* of the cyclist, which can be male or female... apparently it matters :neutral_face: 

It appears to work by appending the gender unicode character after the cyclist character i.e., 🚴 + ♂️ = 🚴‍♂️ (person-biking + male-sign = man-biking) but whether you see the difference or not depends on the platform

Reply:
🤔🧐